src/index.js

```diff
@@ -80,6 +80,11 @@ function parseArgs(providerName) {
   return parser.parseArgs();
 }
 
+function getSelectedAccount(args) {
```

Comment:
`config` is in the global scope, but args is not, I could also pass config as a parameter so it is consistent. 

Since this function creates an account object I don't want to mix its responsibility with the `co` function. There is nothing wrong by having a single caller or two lines, it improves readability and follows clean code practices.

Reply:
"## If a function description includes an "and", it's wrong
Functions should do one thing and one thing only. When you're writing the function documentation and find that you added an "and", it means the function is doing more than one thing. Break that function into two and remove the "and"." - Julio Biason

Reply:
removed getSelectedAccount